receiver/hostmetricsreceiver/internal/scraper.go

```diff
@@ -46,29 +48,13 @@ type Factory interface {
 	CreateMetricsScraper(
 		ctx context.Context,
 		logger *zap.Logger,
-		cfg Config,
-		consumer consumer.MetricsConsumer) (Scraper, error)
+		cfg Config) (Scraper, error)
 }
 
 // Config is the configuration of a scraper.
 type Config interface {
-	// CollectionInterval returns the interval at which the scraper collects metrics
-	CollectionInterval() time.Duration
-	// SetCollectionInterval sets the interval at which the scraper collects metrics
-	SetCollectionInterval(time.Duration)
 }
 
 // ConfigSettings provides common settings for scraper configuration.
 type ConfigSettings struct {
```

Comment:
Note I didn't remove this now empty struct as it will be used in the near future to support default configuration for enabling/disabling individual metrics by name